app/models/repository.rb

```diff
@@ -20,6 +20,10 @@ def self.license_options
      "Attribution - Non-Commercial - No Derivatives"]
   end
 
+  def user_username
```

Comment:
because for some reason when I give the first repository fixture `user_username: bob` the following test fails:
`test "Uniqueness of repository title" do 
  repository = repositories(:two)  
  repository.title = "Repository1"
  assert repository.invalid? , "Project title is already in use."` 

even though there's a `user_username` column in Repository. Adding this seem to pass the test as well as render views properly